core/notifications/src/email_executor/smtp/config.rs

```diff
@@ -2,10 +2,14 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Default, Debug, Deserialize, Serialize)]
 pub struct SmtpConfig {
+    #[serde(default)]
     pub username: String,
     #[serde(default)]
     pub password: String,
+    #[serde(default)]
     pub from_email: String,
+    #[serde(default)]
     pub relay: String,
+    #[serde(default)]
```

Comment:
This is not a reasonable default or?

Reply:
you mean i should create impl default for port ?